### ISSUE TITLE
Accuracy starts from 0 when fine-tuning a pre-trained model

### ISSUE THREAD

**Issue by arsaey** [PaddlePaddle/PaddleOCR#14705]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

I am trying to fine-tune a PaddleOCR model. I extracted text from images using PaddleOCR and made minor manual corrections to a few characters. When I start fine-tuning with the following command:

python3 tools/train.py -c ./ar_train_2/config.yml -o Global.pretrained_model=./ar_train_2/best_accuracy

The accuracy starts from 0, which is unexpected because the model was already trained.

What I Have Checked

✅ The pre-trained model path is correct and exists. 
✅ The model loads properly (paddle.load() confirms this).
✅ The checkpoints option also works correctly.
✅ The architecture in config.yml has not changed. (just increase epoch_num -   use_gpu: false - and path to data - checkpoints... 
✅ No errors appear in the logs indicating failure to load weights.

Expected Behavior
The accuracy should not start from 0, and the model should continue learning from its previous state.

### 🏃‍♂️ Environment (运行环境)

os ubuntu

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

if you need it let me know

dictionary is correct also

and this is my log

[2025/02/18 03:22:53] ppocr WARNING: Skipping import of the encryption module.
[2025/02/18 03:22:53] ppocr INFO: Architecture : 
[2025/02/18 03:22:53] ppocr INFO:     Backbone : 
[2025/02/18 03:22:53] ppocr INFO:         last_conv_stride : [1, 2]
[2025/02/18 03:22:53] ppocr INFO:         last_pool_type : avg
[2025/02/18 03:22:53] ppocr INFO:         name : MobileNetV1Enhance
[2025/02/18 03:22:53] ppocr INFO:         scale : 0.5
[2025/02/18 03:22:53] ppocr INFO:     Head : 
[2025/02/18 03:22:53] ppocr INFO:         head_list : 
[2025/02/18 03:22:53] ppocr INFO:             CTCHead : 
[2025/02/18 03:22:53] ppocr INFO:                 Head : 
[2025/02/18 03:22:53] ppocr INFO:                     fc_decay : 1e-05
[2025/02/18 03:22:53] ppocr INFO:                 Neck : 
[2025/02/18 03:22:53] ppocr INFO:                     depth : 2
[2025/02/18 03:22:53] ppocr INFO:                     dims : 64
[2025/02/18 03:22:53] ppocr INFO:                     hidden_dims : 120
[2025/02/18 03:22:53] ppocr INFO:                     name : svtr
[2025/02/18 03:22:53] ppocr INFO:                     use_guide : True
[2025/02/18 03:22:53] ppocr INFO:             SARHead : 
[2025/02/18 03:22:53] ppocr INFO:                 enc_dim : 512
[2025/02/18 03:22:53] ppocr INFO:                 max_text_length : 25
[2025/02/18 03:22:53] ppocr INFO:         name : MultiHead
[2025/02/18 03:22:53] ppocr INFO:     Transform : None
[2025/02/18 03:22:53] ppocr INFO:     algorithm : SVTR
[2025/02/18 03:22:53] ppocr INFO:     model_type : rec
[2025/02/18 03:22:53] ppocr INFO: Eval : 
[2025/02/18 03:22:53] ppocr INFO:     dataset : 
[2025/02/18 03:22:53] ppocr INFO:         data_dir : ./data
[2025/02/18 03:22:53] ppocr INFO:         label_file_list : ['./data/train.txt']
[2025/02/18 03:22:53] ppocr INFO:         name : SimpleDataSet
[2025/02/18 03:22:53] ppocr INFO:         transforms : 
[2025/02/18 03:22:53] ppocr INFO:             DecodeImage : 
[2025/02/18 03:22:53] ppocr INFO:                 channel_first : False
[2025/02/18 03:22:53] ppocr INFO:                 img_mode : BGR
[2025/02/18 03:22:53] ppocr INFO:             MultiLabelEncode : None
[2025/02/18 03:22:53] ppocr INFO:             RecResizeImg : 
[2025/02/18 03:22:53] ppocr INFO:                 image_shape : [3, 48, 320]
[2025/02/18 03:22:53] ppocr INFO:             KeepKeys : 
[2025/02/18 03:22:53] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2025/02/18 03:22:53] ppocr INFO:     loader : 
[2025/02/18 03:22:53] ppocr INFO:         batch_size_per_card : 128
[2025/02/18 03:22:53] ppocr INFO:         drop_last : False
[2025/02/18 03:22:53] ppocr INFO:         num_workers : 10
[2025/02/18 03:22:53] ppocr INFO:         shuffle : False
[2025/02/18 03:22:53] ppocr INFO: Global : 
[2025/02/18 03:22:53] ppocr INFO:     cal_metric_during_train : True
[2025/02/18 03:22:53] ppocr INFO:     character_dict_path : ./ppocr/utils/dict/arabic_dict.txt
[2025/02/18 03:22:53] ppocr INFO:     checkpoints : ./ar_train_2/best_accuracy
[2025/02/18 03:22:53] ppocr INFO:     debug : False
[2025/02/18 03:22:53] ppocr INFO:     distributed : False
[2025/02/18 03:22:53] ppocr INFO:     epoch_num : 148
[2025/02/18 03:22:53] ppocr INFO:     eval_batch_step : [0, 2000]
[2025/02/18 03:22:53] ppocr INFO:     infer_mode : False
[2025/02/18 03:22:53] ppocr INFO:     log_smooth_window : 20
[2025/02/18 03:22:53] ppocr INFO:     max_text_length : 25
[2025/02/18 03:22:53] ppocr INFO:     pretrained_model : ./ar_train_2/best_accuracy
[2025/02/18 03:22:53] ppocr INFO:     print_batch_step : 10
[2025/02/18 03:22:53] ppocr INFO:     save_epoch_step : 3
[2025/02/18 03:22:53] ppocr INFO:     save_inference_dir : None
[2025/02/18 03:22:53] ppocr INFO:     save_model_dir : ./persian_v3_final
[2025/02/18 03:22:53] ppocr INFO:     save_res_path : ./output/rec/predicts_ppocrv3_arabic.txt
[2025/02/18 03:22:53] ppocr INFO:     use_gpu : False
[2025/02/18 03:22:53] ppocr INFO:     use_space_char : True
[2025/02/18 03:22:53] ppocr INFO:     use_visualdl : False
[2025/02/18 03:22:53] ppocr INFO: Loss : 
[2025/02/18 03:22:53] ppocr INFO:     loss_config_list : 
[2025/02/18 03:22:53] ppocr INFO:         CTCLoss : None
[2025/02/18 03:22:53] ppocr INFO:         SARLoss : None
[2025/02/18 03:22:53] ppocr INFO:     name : MultiLoss
[2025/02/18 03:22:53] ppocr INFO: Metric : 
[2025/02/18 03:22:53] ppocr INFO:     ignore_space : False
[2025/02/18 03:22:53] ppocr INFO:     main_indicator : acc
[2025/02/18 03:22:53] ppocr INFO:     name : RecMetric
[2025/02/18 03:22:53] ppocr INFO: Optimizer : 
[2025/02/18 03:22:53] ppocr INFO:     beta1 : 0.9
[2025/02/18 03:22:53] ppocr INFO:     beta2 : 0.999
[2025/02/18 03:22:53] ppocr INFO:     lr : 
[2025/02/18 03:22:53] ppocr INFO:         learning_rate : 0.001
[2025/02/18 03:22:53] ppocr INFO:         name : Cosine
[2025/02/18 03:22:53] ppocr INFO:         warmup_epoch : 5
[2025/02/18 03:22:53] ppocr INFO:     name : Adam
[2025/02/18 03:22:53] ppocr INFO:     regularizer : 
[2025/02/18 03:22:53] ppocr INFO:         factor : 3e-05
[2025/02/18 03:22:53] ppocr INFO:         name : L2
[2025/02/18 03:22:53] ppocr INFO: PostProcess : 
[2025/02/18 03:22:53] ppocr INFO:     name : CTCLabelDecode
[2025/02/18 03:22:53] ppocr INFO: Train : 
[2025/02/18 03:22:53] ppocr INFO:     dataset : 
[2025/02/18 03:22:53] ppocr INFO:         data_dir : ./data/
[2025/02/18 03:22:53] ppocr INFO:         ext_op_transform_idx : 1
[2025/02/18 03:22:53] ppocr INFO:         label_file_list : ['./data/train.txt']
[2025/02/18 03:22:53] ppocr INFO:         name : SimpleDataSet
[2025/02/18 03:22:53] ppocr INFO:         transforms : 
[2025/02/18 03:22:53] ppocr INFO:             DecodeImage : 
[2025/02/18 03:22:53] ppocr INFO:                 channel_first : False
[2025/02/18 03:22:53] ppocr INFO:                 img_mode : BGR
[2025/02/18 03:22:53] ppocr INFO:             RecConAug : 
[2025/02/18 03:22:53] ppocr INFO:                 ext_data_num : 2
[2025/02/18 03:22:53] ppocr INFO:                 image_shape : [48, 320, 3]
[2025/02/18 03:22:53] ppocr INFO:                 prob : 0.5
[2025/02/18 03:22:53] ppocr INFO:             RecAug : None
[2025/02/18 03:22:53] ppocr INFO:             MultiLabelEncode : None
[2025/02/18 03:22:53] ppocr INFO:             RecResizeImg : 
[2025/02/18 03:22:53] ppocr INFO:                 image_shape : [3, 48, 320]
[2025/02/18 03:22:53] ppocr INFO:             KeepKeys : 
[2025/02/18 03:22:53] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2025/02/18 03:22:53] ppocr INFO:     loader : 
[2025/02/18 03:22:53] ppocr INFO:         batch_size_per_card : 128
[2025/02/18 03:22:53] ppocr INFO:         drop_last : True
[2025/02/18 03:22:53] ppocr INFO:         num_workers : 10
[2025/02/18 03:22:53] ppocr INFO:         shuffle : True
[2025/02/18 03:22:53] ppocr INFO: profiler_options : None
[2025/02/18 03:22:53] ppocr INFO: train with paddle 2.6.2 and device Place(cpu)
[2025/02/18 03:22:53] ppocr INFO: Initialize indexs of datasets:['./data/train.txt']
[2025/02/18 03:22:53] ppocr INFO: Initialize indexs of datasets:['./data/train.txt']
[2025/02/18 03:22:54] ppocr INFO: train dataloader has 2 iters
[2025/02/18 03:22:54] ppocr INFO: valid dataloader has 3 iters
[2025/02/18 03:22:54] ppocr INFO: resume from ./ar_train_2/best_accuracy
[2025/02/18 03:22:54] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations
[2025/02/18 03:25:31] ppocr INFO: epoch: [98/148], global_step: 2, lr: 0.000025, acc: 0.000000, norm_edit_dis: 0.136419, CTCLoss: 140.938446, SARLoss: 5.346252, loss: 146.284698, avg_reader_cost: 0.30030 s, avg_batch_cost: 15.73456 s, avg_samples: 25.6, ips: 1.62699 samples/s, eta: 2:11:07
[2025/02/18 03:25:32] ppocr INFO: save model in ./persian_v3_final/latest
[2025/02/18 03:28:17] ppocr INFO: epoch: [99/148], global_step: 4, lr: 0.000029, acc: 0.000000, norm_edit_dis: 0.130844, CTCLoss: 138.305687, SARLoss: 5.130360, loss: 143.436043, avg_reader_cost: 0.37283 s, avg_batch_cost: 16.53935 s, avg_samples: 25.6, ips: 1.54782 samples/s, eta: 2:11:47
[2025/02/18 03:28:19] ppocr INFO: save model in ./persian_v3_final/latest
[2025/02/18 03:28:19] ppocr INFO: save model in ./persian_v3_final/iter_epoch_99
[2025/02/18 03:30:52] ppocr INFO: epoch: [100/148], global_step: 6, lr: 0.000028, acc: 0.000000, norm_edit_dis: 0.137362, CTCLoss: 135.424538, SARLoss: 4.836812, loss: 140.261345, avg_reader_cost: 0.53251 s, avg_batch_cost: 15.48373 s, avg_samples: 25.6, ips: 1.65335 samples/s, eta: 2:07:21
[2025/02/18 03:30:54] ppocr INFO: save model in ./persian_v3_final/latest
[2025/02/18 03:33:32] ppocr INFO: epoch: [101/148], global_step: 8, lr: 0.000026, acc: 0.000000, norm_edit_dis: 0.137362, CTCLoss: 131.919357, SARLoss: 4.543669, loss: 136.463020, avg_reader_cost: 0.49639 s, avg_batch_cost: 16.07643 s, avg_samples: 25.6, ips: 1.59239 samples/s, eta: 2:05:00
[2025/02/18 03:33:34] ppocr INFO: save model in ./persian_v3_final/latest
[2025/02/18 03:36:15] ppocr INFO: epoch: [102/148], global_step: 10, lr: 0.000024, acc: 0.000000, norm_edit_dis: 0.141565, CTCLoss: 128.297752, SARLoss: 4.311444, loss: 132.609192, avg_reader_cost: 0.50531 s, avg_batch_cost: 16.23888 s, avg_samples: 25.6, ips: 1.57646 samples/s, eta: 2:02:46
[2025/02/18 03:36:17] ppocr INFO: save model in ./persian_v3_final/latest
[2025/02/18 03:36:18] ppocr INFO: save model in ./persian_v3_final/iter_epoch_102
[2025/02/18 03:38:52] ppocr INFO: epoch: [103/148], global_step: 12, lr: 0.000023, acc: 0.000000, norm_edit_dis: 0.141872, CTCLoss: 125.258484, SARLoss: 4.165138, loss: 129.423622, avg_reader_cost: 0.55775 s, avg_batch_cost: 15.74122 s, avg_samples: 25.6, ips: 1.62630 samples/s, eta: 1:59:46
[2025/02/18 03:38:54] ppocr INFO: save model in ./persian_v3_final/latest

**Comment by arsaey** [comment]
![Image](https://github.com/user-attachments/assets/46bb5a2d-8618-4a8a-9032-04701276e9ba)
![Image](https://github.com/user-attachments/assets/6fae1f49-312f-411c-940e-531b225f8d8c)

there is some sample image of my train data

**Comment by arsaey** [comment]
also i can confirm infer model can read them well

**Comment by jainparas386** [comment]
Hi, I am facing the same issue. Could you please help in resolving it. I have set 500 epochs.
ps: 1.14743 samples/s, eta: 4 days, 13:13:04
eval model:: 100%|█████████████████████████████████████████████████████████████████████████████| 39/39 [00:10<00:00,  3.59it/s]
[2025/02/20 18:22:03] ppocr INFO: cur metric, acc: 0.0, norm_edit_dis: 0.037042441578732266, fps: 7.219528466987536
[2025/02/20 18:22:05] ppocr INFO: save best model is to ./output/v3_en_mobile_multi/best_accuracy
[2025/02/20 18:22:05] ppocr INFO: best metric, acc: 0.0, is_float16: False, norm_edit_dis: 0.037042441578732266, fps: 7.219528466987536, best_epoch: 1
Still getting the acc=0 only.

**Comment by arsaey** [comment]
> Hi, I am facing the same issue. Could you please help in resolving it. I have set 500 epochs. ps: 1.14743 samples/s, eta: 4 days, 13:13:04 eval model:: 100%|█████████████████████████████████████████████████████████████████████████████| 39/39 [00:10<00:00, 3.59it/s] [2025/02/20 18:22:03] ppocr INFO: cur metric, acc: 0.0, norm_edit_dis: 0.037042441578732266, fps: 7.219528466987536 [2025/02/20 18:22:05] ppocr INFO: save best model is to ./output/v3_en_mobile_multi/best_accuracy [2025/02/20 18:22:05] ppocr INFO: best metric, acc: 0.0, is_float16: False, norm_edit_dis: 0.037042441578732266, fps: 7.219528466987536, best_epoch: 1 Still getting the acc=0 only.

which model you try to finetune?

**Comment by arsaey** [comment]
> Hi, I am facing the same issue. Could you please help in resolving it. I have set 500 epochs. ps: 1.14743 samples/s, eta: 4 days, 13:13:04 eval model:: 100%|█████████████████████████████████████████████████████████████████████████████| 39/39 [00:10<00:00, 3.59it/s] [2025/02/20 18:22:03] ppocr INFO: cur metric, acc: 0.0, norm_edit_dis: 0.037042441578732266, fps: 7.219528466987536 [2025/02/20 18:22:05] ppocr INFO: save best model is to ./output/v3_en_mobile_multi/best_accuracy [2025/02/20 18:22:05] ppocr INFO: best metric, acc: 0.0, is_float16: False, norm_edit_dis: 0.037042441578732266, fps: 7.219528466987536, best_epoch: 1 Still getting the acc=0 only.

share your train.txt and config.yml and full log when you start train
